packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -213,24 +213,15 @@ class _NetworkProfilerControlsState extends State<_NetworkProfilerControls>
     final hasRequests = _filteredRequests.isNotEmpty;
     return Row(
       children: [
-        PauseButton(
-          minScreenWidthForTextBeforeScaling:
-              _NetworkProfilerControls._includeTextWidth,
-          tooltip: 'Pause recording network traffic',
+        StartStopRecordingButton(
+          recording: _recording,
+          onPressed: () => widget.controller.togglePolling(!_recording),
+          tooltipOverride: _recording
+              ? 'Stop recording network traffic'
+              : 'Resume recording network traffic',
+          minScreenWidthForTextBeforeScaling: double.infinity,
           gaScreen: gac.network,
           gaSelection: gac.pause,
```

Comment:
the gaSelection also needs a ternary now. 
`gaSelection: recording ? gac.stopRecording : gac:startRecording`

Reply:
Since network and cpu use 2 different sets of selections, I'll leave this at the top level